shared/bash_remediation_functions/service_command.sh

```diff
@@ -47,20 +47,24 @@ else
 fi
 
 # If chkconfig_util is not empty, use chkconfig/service commands.
-if ! [ "x$chkconfig_util" = x ] ; then
+if [ "x$chkconfig_util" != x ] ; then
   $service_util $service $service_operation
   $chkconfig_util --level 0123456 $service $chkconfig_state
 else
   $service_util $service_operation $service
   $service_util $service_state $service
+  # The service may not be running because it has been started and failed,
+  # so let's reset the state so OVAL checks pass.
+  # Service should be 'inactive', not 'failed' after reboot though.
+  $service_util reset-failed $service
```

Comment:
does this work with the "service" command? this looks like something that is systemctl only.

Reply:
The shell is not clear, at first sight, but I believe that the branch of the if statement is the exclusive `systemctl` branch.

Reply:
Haven't tested but does OVAL really report pass/fail on this?

Reply:
@redhatrises The PR #2686 exposes that the OVAL check on whether a service is disabled will fail is the service is indeed disabled, but in a `failed` state. Or you meant something else?

Reply:
@matejak no didn't mean something else... I think I need more coffee as I am confusing the difference between the two PRs.